README.md

```diff
@@ -1,22 +1,20 @@
 # HIPIFY
+HIPIFY is a set of tools to translate CUDA source code into portable [HIP](https://github.com/ROCm-Developer-Tools/HIP) C++ automatically.
 
-### Tools to translate CUDA source code into portable HIP C++ automatically
-## Table of Contents
-
-<!-- toc -->
-
-- [hipify-clang](#clang)
-     * [Dependencies](#dependencies)
-     * [Usage](#hipify-clang-usage)
-     * [Building](#building)
-     * [Testing](#testing)
-     * [Linux](#linux)
-     * [Windows](#windows)
-- [hipify-perl](#perl)
-     * [Usage](#hipify-perl-usage)
-     * [Building](#hipify-perl-building)
-- [Supported CUDA APIs](#cuda-apis)
-- [Disclaimer](#disclaimer)
+## Documentation
+Information about HIPIFY and other user topics is found in the [HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
+The documentation source is browseable [here](docs/source/index.md).
```

Comment:
It looks to me strange, as HIPIFY documentation provided by the link duplicates the README.md itself. Maybe I missed smth?

Reply:
Good point. I have replaced the line about the documentation source with instructions to build the docs locally.
[6f7acbb](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/6f7acbb5362f921fb477393e4b770dd017cc9e4d)